packages/shell/src/ui/shell-ui-layout.tsx

```diff
@@ -33,7 +35,11 @@ export function ShellUiLayout({ links }: { links: ShellLayoutLink[] }) {
             to={to}
           >
             <UiIcon icon={icon} />
-            {label}
+            {/* TODO: Figure out how i18next-cli can extract a key like $[label] */}
+            {label === 'Portfolio' ? t(($) => $.portfolioLabel) : null}
+            {label === 'Explorer' ? t(($) => $.explorerLabel) : null}
+            {label === 'Tools' ? t(($) => $.toolsLabel) : null}
+            {label === 'Settings' ? t(($) => $.settingsLabel) : null}
```

Comment:
@beeman Not sure if you have any suggestions here. I don't like it but could not find a better way for i18next-cli to work

Reply:
I think we should do it like this:

```ts
export function useShellLayoutLinks(): ShellLayoutLink[] {
  const { t } = useTranslation('shell')
  return [
    { icon: 'portfolio', label: t(($) => $.labelPortfolio), to: '/portfolio' },
    { icon: 'explorer', label: t(($) => $.labelExplorer), to: '/explorer' },
    { icon: 'tools', label: t(($) => $.labelTools), to: '/tools' },
    { icon: 'settings', label: t(($) => $.labelSettings), to: '/settings' },
  ]
}
```

For example, i18n was the reason I popped [these ones](https://github.com/samui-build/samui-wallet/blob/1b2811f41165a929e0f7754d7e39e7c60d08124e/packages/settings/src/data-access/use-settings-pages.tsx) in a hook.